Remove the list of overridden functions; it is better to include socks.h.

--- a/doc/README.usage
+++ b/doc/README.usage
@@ -27,29 +27,8 @@
 is then linked with the libsocks library in the distribution (-lsocksd,
 not -ldsocksd).
 
-These function calls can be found in the socks library:
-Raccept
-Rbind 
-Rbindresvport
-Rconnect
-Rgetaddrinfo()
-Rgethostbyname
-Rgethostbyname2
-Rgetipnodebyname()
-Rgetpeername
-Rgetsockname
-Rread
-Rreadv
-Rrecv
-Rrecvfrom
-Rrecvfrom
-Rrecvmsg
-Rrresvport
-Rsend
-Rsendmsg
-Rsendto
-Rwrite
-Rwritev
+These system calls can be found in the socks include file "socks.h"
+which should be added to all relevant source files.
 
 These calls currently does nothing and are only provided for
 compatibility with programs which use them:
@@ -57,16 +36,6 @@
 Rselect
 SOCKSinit
 
-The source can either be modified directly, or by using defines to
-change the source during compilation, as following:
-
-Either add this line to the command compilation line (e.g. CFLAGS):
--Dconnect=Rconnect -Dbind=Rbind -Dgetsockname=Rgetsockname -Dgetpeername=Rgetpeername -Daccept=Raccept -Drresvport=Rrresvport -Dbindresvport=Rbindresvport -Dgethostbyname=Rgethostbyname -Dgethostbyname2=Rgethostbyname2 -Dsendto=Rsendto -Drecvfrom=Rrecvfrom -Drecvfrom=Rrecvfrom -Dwrite=Rwrite -Dwritev=Rwritev -Dsend=Rsend -Dsendmsg=Rsendmsg -Dread=Rread -Dreadv=Rreadv -Drecv=Rrecv -Drecvmsg=Rrecvmsg -Dgetaddrinfo=Rgetaddrinfo -Dgetipnodebyname=Rgetipnodebyname
-
-or add "include <socks.h>" to every C file in the program you are
-compiling.
-
-
 
 Libraries
 	In total there are three libraries distributed with this package:
